### DISCUSSION TITLE
When TIN/EIN is not provided....

### DISCUSSION THREAD

**Discussion by caracolina** [CMSgov/price-transparency-guide#522] [Q&A]
We don't have EINs for the provider references so there will be a 1-to-1 relationship in the provider reference arrays. We were wondering though if it is possible to place a "placeholder" TIN into the TIN-identifier field and that would allow us to add all appropriate NPIs into the NPI array portion.

Example:
"provider_groups": [{
"npi": [1111111111, 2222222222, 3333333333, 4444444444, 5555555555],
"tin":{
"type": "ein",
"value": "00-0000000"
}]

It would help us out and make the files smaller.

Thank you for your feedback,
Caroline

**Comment by shaselton-usds** [comment]
Hi @caracolina -- Just want to clarify, you don't have the EIN in the contract (or associated to the contract) for the item/service?

**Reply by riashah28** [reply]
Hi @shaselton-usds - following up on this. We are seeing the same issue so are using the same NPI in the TIN object and provider object. Can you please confirm if this works?

**Comment by caracolina** [comment]
Our client can't guarantee that there are no SSNs in the EIN field, so they decided to supply only NPIs. 

Thank you,
Caroline

**Comment by abd-hh** [comment]
@shaselton-usds bump on this
